collect_app/src/main/java/org/odk/collect/android/audio/AudioControllerView.java

```diff
@@ -193,10 +210,20 @@ public void onStopTrackingTouch(SeekBar seekBar) {
                 listener.onPlayClicked();
                 wasPlaying = false;
             }
+
+            if (analytics != null) {
+                analytics.onSeek();
+            }
         }
 
         Boolean isSwiping() {
             return swiping;
         }
     }
+
+    public interface AnalyticsListener {
```

Comment:
Felt less intrusive just to create a separate listener here rather than adding (and kind of duplicating) methods to the main listener.